shaders/shader.rchit

```diff
@@ -1,5 +1,6 @@
 #version 460
 #extension GL_EXT_ray_tracing : require
+#extension GL_GOOGLE_include_directive : require
```

Comment:
Can you explain this `GL_GOOGLE_include_directive` to me?

Reply:
This is needed for ```#include```, or that compiler will report error for ```#include``` directives.